cli/cmd/version.go

```diff
@@ -22,3 +26,49 @@ var versionCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var availableVersionCmd = &cobra.Command{
+	Use:          "available-version",
+	Short:        "Latest available version of Happy",
+	Long:         "Returns the latest available version of Happy",
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		v, err := GetLatestAvailableVersion(cmd)
+		if err != nil {
+			fmt.Fprint(cmd.Parent().ErrOrStderr(), err)
+			return err
+		}
+
+		fmt.Fprint(cmd.OutOrStdout(), v)
+		return nil
+	},
+}
+
+func GetLatestAvailableVersion(cmd *cobra.Command) (*util.Release, error) {
+	happyConfig, err := config.GetHappyConfigForCmd(cmd)
+	if err != nil {
+		return nil, err
+	}
+
+	api := hapi.MakeApiClient(happyConfig)
+	result := model.HealthResponse{}
+	err = api.GetParsed("/health", "", &result)
+	if err != nil {
+		return nil, err
+	}
+
+	return &util.Release{
+		Version: result.Version,
```

Comment:
Also, just to confirm. The version that comes back from the API, this is the CLI version? Not the API version right?

Reply:
jk, I just checked it out. These should be the same

Reply:
Good point, and thanks for checking!